app/serializers/api/public/v1/serializable_location.rb

```diff
@@ -0,0 +1,41 @@
+module API
+  module Public
+    module V1
+      class SerializableLocation < JSONAPI::Serializable::Resource
+        extend JSONAPI::Serializable::Resource::ConditionalFields
+
+        type "locations"
+
+        belongs_to :recruitment_cycle
+        belongs_to :provider
+
+        belongs_to :course, unless: -> { @course.nil? } do
+          data { @course }
+        end
+
+        attributes :code,
+                   :latitude,
+                   :location_name,
```

Comment:
In the API docs, `location_name` is `name`.

`latitude` and `longitude` are potentially tricky because Google, iirc, doesn't allow for them to be sent like this. Might be worth a discussion.

Also, while I'm on here, need to update the docs to remove the `recruitment_cycle_year`.

Reply:
done in https://github.com/DFE-Digital/teacher-training-api/pull/1513/commits/4782be155bacb6936b89c83af81270f11e14aff5 and https://github.com/DFE-Digital/teacher-training-api/pull/1513/commits/733836fcade0b1e65f7f05c559bde835dd30b9ba